Sources/Basic/Process.swift

```diff
@@ -202,7 +205,7 @@ public final class Process: ObjectIdentifierProtocol {
     ///
     /// Key: Executable name or path.
     /// Value: Path to the executable, if found.
-    static private var validatedExecutablesMap = [String: AbsolutePath?]()
+    static var validatedExecutablesMap = [String: AbsolutePath?]()
```

Comment:
We should make this `private(set)` and add a comment "// FIXME: @testable(internal)".